spec/features/cart/checkout_spec.rb

```diff
@@ -27,6 +27,43 @@
 
       expect(current_path).to eq("/orders/new")
     end
+
```

Comment:
Test that cart is empty after order is placed and a flash message tells the user the order was placed